taskfiles/Taskfile_gotest.yml

```diff
@@ -62,6 +60,4 @@ tasks:
             -v \
             -coverprofile={{.UNITTEST_DIR}}/main.out \
             ./... \
-            2>&1 > {{.UNITTEST_DIR}}/main.out
-
-          # gocov convert {{.UNITTEST_DIR}}/main.out > {{.UNITTEST_DIR}}/coverage.json
\ No newline at end of file
+            2>&1 | go-junit-report -set-exit-code -debug.print-events > {{.UNITTEST_DIR}}/main.xml
```

Comment:
are you sure that go-junit-report is avaialable for regular development ? If you leave this change out of the PR I can easily tick

Reply:
done